iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -159,25 +160,20 @@ public static Collection inputs() throws Exception
     // How much to wait until a message makes it to the server, in milliseconds
     protected static final Integer SEND_TIMEOUT_MILLISECONDS = 60000;
 
-    public enum STATUS
-    {
-        SUCCESS, FAILURE, UNKNOWN
-    }
-
     public class DeviceTwinStatusCallBack implements IotHubEventCallback
     {
         public void execute(IotHubStatusCode status, Object context)
         {
             DeviceState state = (DeviceState) context;
 
-            //On failure, Don't update status any further
```

Comment:
how is this going to affect the callback? is this considered a breaking change?

Reply:
This is in the test code, so it isn't a breaking change, no

Reply:
ah, the file path threw me off